### PR TITLE
Kubemark changes.

### PR DESCRIPTION
**Pull request by shivramsrivastava**

### CODE REVIEW

File: cluster/gce/config-common.sh [hunk 1]
```diff
@@ -28,13 +28,13 @@ function get-master-size {
     suggested_master_size=8
   fi
   if [[ "${NUM_NODES}" -gt "250" ]]; then
-    suggested_master_size=16
+    suggested_master_size=8
```
Comment (wojtek-t) [inline cluster/gce/config-common.sh]: No - all of them are not what we recommend. Please revert.

File: cluster/gce/config-default.sh [hunk 2]
```diff
@@ -31,7 +31,9 @@ NODE_SIZE=${NODE_SIZE:-n1-standard-2}
 NUM_NODES=${NUM_NODES:-3}
 MASTER_SIZE=${MASTER_SIZE:-n1-standard-$(get-master-size)}
 MASTER_DISK_TYPE=pd-ssd
-MASTER_DISK_SIZE=${MASTER_DISK_SIZE:-$(get-master-disk-size)}
+# Kubemark_changes: Hardcoded disk size to 20GB
```
Comment (wojtek-t) [inline cluster/gce/config-default.sh]: That's not what we want - please revert.

File: cluster/gce/config-default.sh [hunk 3]
```diff
@@ -73,11 +75,16 @@ fi
 # Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
 CVM_VERSION=${CVM_VERSION:-container-vm-v20170627}
-GCI_VERSION=${KUBE_GCI_VERSION:-cos-stable-59-9460-64-0}
+# Kubemark_changes: Changed kubemark master image and project.
+#GCI_VERSION=${KUBE_GCI_VERSION:-cos-stable-59-9460-64-0}
```
Comment (wojtek-t) [inline cluster/gce/config-default.sh]: Same here - if you want to use ubuntu - you can always overwrite. Please revert.

File: cluster/kubemark/gce/config-default.sh [hunk 4]
```diff
@@ -25,21 +25,27 @@ GCLOUD=gcloud
...
+# Kubemark_changes: Changed master root disk size to 20 GB.
+MASTER_ROOT_DISK_SIZE=${KUBEMARK_MASTER_ROOT_DISK_SIZE:-20GB}
 REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-false}
 PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 
-MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-gci}
-NODE_OS_DISTRIBUTION=${KUBE_NODE_OS_DISTRIBUTION:-gci}
-MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-cos-stable-59-9460-64-0}
-MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-cos-cloud}
+MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-debian}
```
Comment (wojtek-t) [inline cluster/kubemark/gce/config-default.sh]: Why are you doing those changes. We are deprecating debian.

**Review by wojtek-t — COMMENTED** [review]
Sorry - I don't understand those changes. If you want anything like that, please open an issue to discuss it before creating a PR.

**Comment by shivramsrivastava** [comment]
Sorry for the trouble and confusion. By mistake, pull request was raised.

**Pull request closed without merging**